src/inlined.h

```diff
@@ -55,6 +55,41 @@ static inline uint32_t simple_uhash(const char *name) {
 
 #endif /* HAVE_STMT_EXPR */
 
+static inline long str2l(const char *s) {
+    register long n = 0;
+    register char c, negative = (*s == '-');
```

Comment:
The `register` keywords can be eliminated. AFAIK, no compiler has done anything useful with them in ages. And the `negative` variable should probably be type `bool`.